ctx.go

```diff
@@ -1819,20 +1819,26 @@ func (c *DefaultCtx) configDependentPaths() {
 }
 
 // IsProxyTrusted checks trustworthiness of remote ip.
-// If EnableTrustedProxyCheck false, it returns true
+// If Config.TrustProxy false, it returns true
 // IsProxyTrusted can check remote ip by proxy ranges and ip map.
 func (c *DefaultCtx) IsProxyTrusted() bool {
-	if !c.app.config.EnableTrustedProxyCheck {
+	if !c.app.config.TrustProxy {
 		return true
 	}
 
 	ip := c.fasthttp.RemoteIP()
 
-	if _, trusted := c.app.config.trustedProxiesMap[ip.String()]; trusted {
+	if (c.app.config.TrustProxyConfig.Private && ip.IsPrivate()) ||
+		(c.app.config.TrustProxyConfig.Loopback && ip.IsLoopback()) ||
```

Comment:
Move loopback to the beginning of the IF statement. That way it short-circuits. It's very likely most traffic will be proxied from loopback so.
1. Loopback
2. Private
3. LinkLocal